packages/authorization-process/test/getClientById.test.ts

```diff
@@ -0,0 +1,41 @@
+import { genericLogger } from "pagopa-interop-commons";
+import { Client, generateId, unsafeBrandId } from "pagopa-interop-models";
+import { describe, it, expect } from "vitest";
+import { clientNotFound } from "../src/model/domain/errors.js";
+import { addOneClient, authorizationService, getMockClient } from "./utils.js";
+
+describe("getClientById", async () => {
+  it("should get from the readModel the client with the secified Id with users", async () => {
+    const organizationId = generateId();
+    const expectedClient: Client = {
+      ...getMockClient(),
+      consumerId: unsafeBrandId(organizationId),
+    };
+    await addOneClient(expectedClient);
+
+    const { client } = await authorizationService.getClientById(
+      expectedClient.id,
+      unsafeBrandId(organizationId),
+      genericLogger
+    );
+    expect(client).toMatchObject(expectedClient);
+  });
+  it("should get from the readModel the client with the secified Id without users", async () => {
+    const mockClient = getMockClient();
+
+    await addOneClient(mockClient);
+
+    const { client } = await authorizationService.getClientById(
+      mockClient.id,
+      generateId(),
+      genericLogger
+    );
+    expect(client).toMatchObject(mockClient);
+  });
+  it("should throw clientNotFound if the client with the specified Id doesn't exist", async () => {
+    const client = getMockClient();
+    await expect(
+      authorizationService.getClientById(client.id, generateId(), genericLogger)
+    ).rejects.toThrowError(clientNotFound(client.id));
+  });
+});
```

Comment:
To make the test more meaningful, could you add a different client in readmodel? So that the filter is actually tested.